ydb/public/lib/ydb_cli/commands/ydb_service_topic.cpp

```diff
@@ -228,7 +224,12 @@ namespace {
         settings.PartitioningSettings(PartitionsCount_, PartitionsCount_);
         settings.PartitionWriteBurstBytes(PartitionWriteSpeedKbps_ * 1_KB);
         settings.PartitionWriteSpeedBytesPerSecond(PartitionWriteSpeedKbps_ * 1_KB);
-        settings.SetSupportedCodecs(GetCodecs());
+
+        auto codecs = GetCodecs();
+        if (codecs.empty()) {
```

Comment:
No way. No codecs, no setting. Do not set SupportedCodecs in settings and that's all.